src/main/scala/org/broadinstitute/hail/io/annotators/SampleFamAnnotator.scala

```diff
@@ -12,17 +12,24 @@ object SampleFamAnnotator {
   val numericRegex =
     """^-?(?:\d+|\d*\.\d+)(?:[eE]-?\d+)?$""".r
 
-  def apply(filename: String, delim: String, isQuantitative: Boolean, missing: String,
+  val famID = "famID"
```

Comment:
The reason for moving them out is so I can use them when exporting the fam file and be sure that they stay consistent. Because these are Hail defaults, I'd think it's better to have them as accessible values rather than hardcoded in different places.